index.html

```diff
@@ -165,6 +165,21 @@ <h2>Skills</h2>
         </li>
       </ul>
       </section>
+      <section class="form">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form name="form1" action="https://formspree.io/f/myyovzdk" method="post">
+          <label >
+            <input type="text" name="name" id="name"  placeholder="Yeremias" maxlength="30" required>
+          </label> <br>
+          <label >
+           <input type="email" name="email-address" id="email" placeholder="natajayanj14@gmail.com" maxlength="500" required>
+          </label>
+          <textarea name="comment" rows="8" cols="80" placeholder="Write your message here" required></textarea>
```

Comment:
- [ ] Good job so far. I think the last reviewer mentioned the wrong tag last time. The correct way should be for both inputs should have `maxlength="30"` and the texture should have `maxlength="500"`
- [ ] Please also change background color to `#6070FF`. Good luck 🍀🙌🏼

Your design
![image](https://user-images.githubusercontent.com/69011963/172435197-d199dd92-48bb-4e97-b4e0-10ca97c9bde8.png)

Original design (ignore the gray line please ;) )
![image](https://user-images.githubusercontent.com/69011963/172435418-e32194e3-c3de-4ebd-b602-cfd8ead38d72.png)